tests/framework/hybridModel/test_logical_code.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/hybridModel.logicalCode</name>
+    <author>wangc</author>
+    <created>2020-05-07</created>
+    <classesTested>Models.LogicalModel</classesTested>
+    <description>
+      Example of usage of the Logical Model capability in RAVEN
+      This test is intended to check the functionality of Logical Model
+      with the generic code system.
```

Comment:
can you be more descriptive here? What is the control function? and the models? sorry i know it is tedious but it can save us time later for updating the tests description

Reply:
updated